packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -796,11 +804,17 @@ class Scaffold extends StatefulWidget {
     this.resizeToAvoidBottomPadding,
     this.resizeToAvoidBottomInset,
     this.primary = true,
+    this.extendedBody = false,
     this.drawerDragStartBehavior = DragStartBehavior.down,
   }) : assert(primary != null),
        assert(drawerDragStartBehavior != null),
        super(key: key);
 
+  /// If true and a [bottomNavigationBar] is specified then the body
+  /// extends to the bottom of the Scaffold, instead of only extending
+  /// to the top of the [bottomNavigationBar].
+  final bool extendedBody;
```

Comment:
this should probably be called "extendBody" rather than "extendedBody".

"extend body" says whether we should extend the body or not.
"extended body" says whether the body is extended or not.

In this case, the body can be either, it doesn't have an intrinsic property of being extended. We decide whether to extend it or not. Hence "extend body".

Reply:
renamed param to "extendBody" 17f794a